controller/api/public/stat_summary.go

```diff
@@ -0,0 +1,334 @@
+package public
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/prometheus/common/model"
+	apiUtil "github.com/runconduit/conduit/controller/api/util"
+	pb "github.com/runconduit/conduit/controller/gen/public"
+	"github.com/runconduit/conduit/pkg/k8s"
+	log "github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	reqQuery = "sum(increase(response_total{%s}[%s])) by (%s, classification)"
```

Comment:
we may want to consider this for a subsequent PR, but we can rely on prometheus for success rate calculations:
```
sum(irate(response_total{direction = "inbound", classification="success"}[1m])) by (deployment)
/
sum(irate(response_total{direction = "inbound"}[1m])) by (deployment)
```

related, i'm seeing 100% SR via CLI, but (i think more correct?) ~80% SR when querying prometheus directly:
```bash
$ go run cli/main.go statsummary deployment -a emojivoto
NAME       MESHED   IN_RPS   IN_SUCCESS   IN_LATENCY_P50   IN_LATENCY_P99
emoji         1/1   2.1rps      100.00%              0ms              0ms
vote-bot      1/1   0.1rps      100.00%              0ms              0ms
voting        1/1   0.9rps      100.00%              0ms              0ms
web           1/1   1.9rps      100.00%              0ms              0ms
```

![screen shot 2018-04-04 at 6 59 39 pm](https://user-images.githubusercontent.com/236915/38343714-5ed9d64e-383a-11e8-8104-3312392edaed.png)

Reply:
made #687 to track switching to prometheus. 
the SRs you're seeing are fishy - I'll investigate